src/pytestqt/qtbot.py

```diff
@@ -725,13 +776,18 @@ def mouseRelease(*args, **kwargs):
 
 
 # provide easy access to exceptions to qtbot fixtures
-QtBot.SignalEmittedError = SignalEmittedError
-QtBot.TimeoutError = TimeoutError
-QtBot.ScreenshotError = ScreenshotError
-QtBot.CallbackCalledTwiceError = CallbackCalledTwiceError
-
-
-def _add_widget(item, widget, *, before_close_func=None):
+QtBot.SignalEmittedError = SignalEmittedError  # type: ignore[attr-defined]
```

Comment:
Does using `ignore[attr-defined]` here means that the type checker does not know about `QtBot.SignalEmittedError`?

If so, we should move this to the class declaration:

```python
class QtBot:
    ...
    SignalEmittedError: ClassVar[Type[SignalEmittedError]] = SignalEmittedError
```

(Same for the other exceptions of course)

Reply:
Thanks for spotting, yes code completion seems to not see it

If they are class vars with assignment directly, type inference seems to suffice